src/views/Viewer.vue

```diff
@@ -926,7 +926,10 @@ export default {
 					},
 					iconSvgInline: () => EyeSvg,
 					default: DefaultType.DEFAULT,
-					enabled: (nodes) => {
+					enabled: (nodes, view) => {
+						if (view.id === 'trashbin') {
+							return false
+						}
```

Comment:
Really not fond of this, what if a new view gets added that also shouldn't display Files? We'll have to update this again :)

Maybe we could, if it's a dav ressource, check if th file is at the root of the user like we do in other locations?
Something like this? Not sure with the double negation, please double check :see_no_evil: 

```suggestion
					enabled: (nodes, view) => {
						if (!nodes.some(node => !(node.isDavRessource && node.root?.startsWith('/files'))) {
							return false
						}
```